units/systemd-journal-gatewayd.service.in

```diff
@@ -30,9 +30,9 @@ RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
 SystemCallArchitectures=native
 LockPersonality=yes
 
-# If there are many split upjournal files we need a lot of fds to
-# access them all and combine
-LimitNOFILE=16384
+# If there are many split up journal files we need a lot of fds to access them
+# all and combine them.
+LimitNOFILE=262144
```

Comment:
I'd prefer to see the HIGH_RLIMIT_NOFILE number exported as a configuration variable (with a fixed value, not configurable) and inserted here.